src/components/app.tsx

```diff
@@ -213,95 +207,119 @@ export class App extends React.Component<AppProps, AppState> {
         this.setState({ sensorSlots });
     }
 
-    handleZeroSensor = (sensorSlot:SensorSlot, sensorValue:number) => {
-        sensorSlot.sensor.tareValue = sensorValue;
-        this.setState({ sensorSlots: this.state.sensorSlots });
-    }
-    
     sensorHasData():boolean {
```

Comment:
Consider `hasSensorData()`
- `sensorHasData()` reads to me like a question about a single sensor, while `hasSensorData()` reads more like a question about whether sensor data is available. (May just be me.)
- I generally try to follow the convention that boolean accessors should start with the question, e.g. `is...()`, `has...()`.